Documentation/project-docs/developer-guide.md

```diff
@@ -248,15 +256,62 @@ When running tests by building a test project, tests that don't apply to the `OS
 ```sh
 <repo-root>/Tools/msbuild.sh <csproj_file> /t:BuildAndTest /p:OSGroup=Linux
 ```
-
-_**`ActiveIssue(int issue, TestPlatforms platforms)`:**_
-Use this attribute over test methods to skip failing tests only on the specific platforms; if no platforms are specified, then the test is skipped on all platforms. This attribute returns the 'failing' category, which is disabled by default.
-
 To run all Linux-compatible tests that are failing:
 ```sh
 <repo-root>/Tools/msbuild.sh <csproj_file> /t:BuildAndTest /p:OSGroup=Linux /p:WithCategories=failing
 ```
 
+#### ActiveIssueAttribute
+This attribute is intended to be used when there is an active issue tracking the test failure and it is needed to be fixed. This is a temporary attribute to skip the test while the issue is fixed.
```

Comment:
.. It is important that you limit the scope of the attribute to just the platforms and target monikers where the issue applies.